consensus/src/epoch_manager.rs

```diff
@@ -836,6 +846,29 @@ impl EpochManager {
         Ok(None)
     }
 
+    async fn check_quorum_store_enabled(
```

Comment:
The name is confusing. Maybe check_unverified_event_type?

Reply:
this is not an async function, also isn't this redundant since we already pass `self.quorum_store_enabled` to verify function

Reply:
I originally did the check in the verify function, but changed it to do the separate check based on Sasha's feedback (https://github.com/aptos-labs/aptos-core/pull/5779#discussion_r1043991483). I think this is a bit clearer; it avoids dirtying the verify function which should really verify the correctness of the message.